x-pack/plugins/case/server/routes/api/__tests__/get_case.test.ts

```diff
@@ -37,8 +39,13 @@ describe('GET case', () => {
     const response = await routeHandler(theContext, request, kibanaResponseFactory);
 
     expect(response.status).toEqual(200);
-    expect(response.payload).toEqual(mockCases.find(s => s.id === 'mock-id-1'));
-    expect(response.payload.comments).toBeUndefined();
+    expect(response.payload).toEqual(
+      flattenCaseSavedObject(
+        (mockCases.find(s => s.id === 'mock-id-1') as unknown) as SavedObject<CaseAttributes>,
```

Comment:
maybe we should define our mockCases: SavedObject<CaseAttributes> to avoid a cast here.

Reply:
<img width="673" alt="Screen Shot 2020-02-13 at 9 48 35 AM" src="https://user-images.githubusercontent.com/6935300/74457834-095d2d00-4e46-11ea-99ba-0890d1d35873.png">
doing that displeases the type gods :(